frontend/mgramseva/lib/screeens/consumer_details/consumer_details.dart

```diff
@@ -265,26 +267,29 @@ class _ConsumerDetailsState extends State<ConsumerDetails> {
                           ),
 
                           //Consumer Old Connection Field
-                          Consumer<ConsumerProvider>(
-                            builder: (_, consumerProvider, child) =>
-                                BuildTextField(
-                              i18.consumer.OLD_CONNECTION_ID,
-                              consumerProvider
-                                  .waterconnection.OldConnectionCtrl,
-                              validator: (val) =>
-                                  Validators.maxCharactersValidator(
-                                      val, 20, i18.consumer.OLD_CONNECTION_ID),
-                              isRequired: true,
-                              contextKey:
-                                  consumerProvider.consmerWalkthrougList[4].key,
-                              key: Keys.createConsumer.CONSUMER_OLD_ID_KEY,
-                              readOnly: consumerProvider.isEdit ==
-                                  true,
-                              inputFormatter: [
-                                FilteringTextInputFormatter.allow(
-                                    RegExp("[a-zA-Z0-9/\\-]"))
-                              ],
-                              isDisabled: false,
+                          Visibility(
```

Comment:
Understood that concept but as of know we are doing some changes and we are not required that component know in future we need to show them to user so instead of remove that elements I used visibility so, in future if need then we can show them